ciphers/affine_cipher.py

```diff
@@ -3,32 +3,12 @@
 
 from . import cryptomath_module as cryptomath
 
-SYMBOLS = (
+SYMBOLS: str = (
```

Comment:
If mypy does not need a type, then let's not add one because it creates unnecessary visual clutter.  Python, mypy, and the reader are able to determine from the assignment that `SYMBOLS` is a `str`.  For mypy, the proof of this can be seen in:
```
s = "five"
# s = 5
```
If you mypy the code as is the tests pass.
If you remove the # comment from the second line, mypy says:
`error: Incompatible types in assignment (expression has type "int", variable has type "str")`

Reply:
Yes, that makes sense! Thanks for pointing it out.